lib/v2/screens/profile_screens/citizenship/citizenship_screen.dart

```diff
@@ -0,0 +1,195 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:seeds/v2/components/profile_avatar.dart';
+import 'package:seeds/design/app_theme.dart';
+import 'package:seeds/constants/app_colors.dart';
+import 'package:seeds/v2/domain-shared/ui_constants.dart';
+import 'package:step_progress_indicator/step_progress_indicator.dart';
+import 'package:seeds/v2/screens/profile_screens/citizenship/components/citizenship_progress_item.dart';
+import 'package:seeds/i18n/citizenship.18n.dart';
+
+/// CITIZENSHIP SCREEN
+class CitizenshipScreen extends StatefulWidget {
+  const CitizenshipScreen();
+
+  @override
+  _CitizenshipScreenState createState() => _CitizenshipScreenState();
+}
+
+class _CitizenshipScreenState extends State<CitizenshipScreen> with TickerProviderStateMixin {
+  AnimationController _controller;
+  Animation<double> _timeLineAnimation,
+      _reputationAnimation,
+      _visitorsAnimation,
+      _ageAnimation,
+      _seedsAnimation,
+      _transactionsAnimation,
+      _friendsAnimation;
+  int _timeLine = 0, _reputation = 0, _visitors = 0, _age = 0, _seeds = 0, _transactions = 0, _friends = 0;
```

Comment:
these should live in the view model AKA state

Reply:
Yeah, I already know. What happens is that I did not want to make the PR bigger